src/App.tsx

```diff
@@ -20,9 +21,9 @@ export const App: React.FC<{}> = () => (
             <Route exact path="/">
               <HomePage />
             </Route>
-            {ITProfessions.map((r) => (
-              <Route exact path={`/${r}`} key={r}>
-                <p>Works {r}</p>
+            {ITProfessions.map((profession) => (
+              <Route exact path={`/details/${profession.route}`} key={profession.route}>
```

Comment:
Typically, we'd use `id` as `key`, because it's should be a unique value - although in this case `route` also meets this requirement.

Reply:
Ok, I've changed it